core/src/test/scala/unit/kafka/server/ReplicaFetchTest.scala

```diff
@@ -18,42 +18,41 @@
 package kafka.server
 
 import scala.collection.Seq
-
-import org.junit.jupiter.api.{AfterEach, BeforeEach, Test, TestInfo}
-import kafka.server.QuorumTestHarness
-import kafka.utils.TestUtils
+import org.junit.jupiter.api.AfterEach
+import kafka.utils.{TestInfoUtils, TestUtils}
 import TestUtils._
+import kafka.integration.KafkaServerTestHarness
 import org.apache.kafka.clients.producer.ProducerRecord
 import org.apache.kafka.common.TopicPartition
 import org.apache.kafka.common.serialization.StringSerializer
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
 
-class ReplicaFetchTest extends QuorumTestHarness  {
-  var brokers: Seq[KafkaServer] = null
+class ReplicaFetchTest extends KafkaServerTestHarness  {
```

Comment:
Perhaps we could extend from `IntegrationTestHarness` instead? Then we just need to override `brokerCount` and we can get rid of the `generateConfigs` override